TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Users/AddUser/Confirm.cshtml

```diff
@@ -11,15 +11,15 @@
 }
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
         <form action="@LinkGenerator.AddUserConfirm(Model.UserId!)" method="post">
             <h1 class="govuk-heading-l">@ViewBag.Title</h1>
 
             <govuk-input asp-for="Email" input-class="govuk-!-width-two-thirds" type="email" disabled />
 
             <govuk-input asp-for="Name" input-class="govuk-!-width-two-thirds" />
 
-            <govuk-radios asp-for="@Model.Role" radios-class="trs-user-roles">
+            <govuk-radios asp-for="@Model.Role" radios-class="trs-user-roles" radios-style="--trs-user-permission-columns: @UserPermissionTypes.All.Count">
```

Comment:
Does this conflict with CSP header?

Reply:
I'm not seeing any errors being shown in any browser. Which is strange, looking at the CSP header I would have expected to see one

Reply:
From Program.cs:
```
app.UseCsp(csp =>
{
    var pageTemplateHelper = app.Services.GetRequiredService<PageTemplateHelper>();

    csp.ByDefaultAllow
        .FromSelf();

    csp.AllowScripts
        .FromSelf()
        .From(pageTemplateHelper.GetCspScriptHashes())
        .AddNonce();

    csp.AllowStyles
        .FromSelf()
        .AllowUnsafeInline();
```